api/app/routers/hfi_calc.py

```diff
@@ -64,6 +67,7 @@ def get_prepared_request(
 
     TODO: give this function a better name.
     """
+    # pylint: disable=too-many-locals
```

Comment:
Is it time to just update our linting settings? I can't say I've improved readability in our code by reducing the local count due to this linting warning - and we end up ignoring it a lot because it just doesn't make sense to reduce the count quite often.

Reply:
No I want to refactor this function, there are too many locals and too much state but I'm not sure now's the time.

Reply:
But if we're open to changing lint settings, can we get rid of the mandatory comments?